ios general/ios general/CreateEmployeeViewController.h

```diff
@@ -0,0 +1,23 @@
+#import <UIKit/UIKit.h>
+#import "MainTableViewController.h"
+
+@protocol AddControllerDelegate
+
+@required
+- (void)showNewEmployee:(id)sender;
```

Comment:
Why is the argument type id, if in fact it is Employee? And why is it called "sender"?